jackal_description/urdf/accessories.urdf.xacro

```diff
@@ -88,11 +88,18 @@
 
   <!-- If enabled, generate the upgraded Navsat payload (a Novatel Smart6). -->
   <xacro:if value="$(optenv JACKAL_NAVSAT 0)">
-    <xacro:include filename="$(find jackal_description)/urdf/accessories/novatel_smart6.urdf.xacro" />
+    <xacro:property name="navsat_model" value="$(optenv JACKAL_NAVSAT_MODEL smart7)" />
```

Comment:
Since Melodic is already released, can this default to the smart6 and for Noetic it can change to the smart7?

Reply:
Because Integration is dealing primarily with Melodic their preference is to update the default to a sensor that's not EOL.  I tend to agree.

Because the underlying driver isn't changing, the actual behaviour of robots already in the wild won't be affected; just the shape of the antenna mounted on the URDF.  Customer support has already OK'd the possibility of support tickets asking about the change.